boards/modalai/fc-v1/src/board_config.h

```diff
@@ -277,8 +277,8 @@
 
 /* Power supply control and monitoring GPIOs */
 
-#define BOARD_NUMBER_BRICKS             0
-#define BOARD_NUMBER_DIGITAL_BRICKS     0
+#define BOARD_NUMBER_BRICKS             (1)
```

Comment:
Could you explain this one?

Reply:
Perhaps a hack but in order to get the vehicle to arm...
https://github.com/PX4/Firmware/blob/4698a09b987428b5d26928935e224a47b827e77e/src/modules/commander/Commander.cpp#L1401
https://github.com/PX4/Firmware/blob/4698a09b987428b5d26928935e224a47b827e77e/src/drivers/adc/ADC.cpp#L218

Reply:
Checking this now to see if there's a better way to do this.